tasks/coverage.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const $ = require('gulp-load-plugins')()
+
+const utils = require('../src/utils')
+const config = require('../config/webpack')
+
+module.exports = {
+  fn (gulp, done) {
+    gulp.task('pre-test', function () {
+      return gulp.src([
+        'src/**/*.js'
+      ]).pipe($.istanbul())
+        .pipe($.istanbul.hookRequire())
+    })
+
+    gulp.task('mocha', ['pre-test'], () => {
+      return gulp.src([
+        'test/node.js',
+        'test/**/*.spec.js'
```

Comment:
> 13:40 <dignifiedquire> no wait the globs for istanbul need to be the same as for mocha so it doesn't load browser.js

This is the same as https://github.com/dignifiedquire/dignified.js/blob/master/tasks/test/node.js#L12-L13

What else would you like me to add?